nix/marlowe-cardano/integration-tests.nix

```diff
@@ -1,13 +1,15 @@
-{ inputs, pkgs, projects }:
+{ inputs', pkgs, l, ... }:
+
+{ projects }:
 
 let
-  cardano-cli = inputs.cardano-world.cardano.packages.cardano-cli;
-  cardano-node = inputs.cardano-world.cardano.packages.cardano-node;
-  marlowe-runtime-cli = projects.ghc8107.hsPkgs.marlowe-runtime-cli.components.exes.marlowe-runtime-cli;
```

Comment:
You can support multiple ghc compilers in your project.
See `./nix/cabal-project.nix:supportedCompilers`, and so `projects` will contain an attribute for each compiler name.
However you can now use the `default` project which will use the `./nix/cabal-project.nix:defaultCompiler` (which is set the first compiler in `supportedCompilers` by default)